src/hooks/useCameraMicrophonePermission.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react'
+
+export const STATE_WAITING = 0
+export const STATE_GRANTED = 1
+export const STATE_DENIED = 2
+
+export default function useCameraMicrophonePermission(name) {
+    const [result, setResult] = React.useState(STATE_WAITING)
+    useEffect(() => {
+        navigator.getUserMedia(
```

Comment:
caution `navigator.getUserMedia` is deprecated use `navigator.mediaDevices.getUserMedia()` instead.
Error on firefox 75
![image](https://user-images.githubusercontent.com/3415615/80806321-a6fd0880-8bba-11ea-9f1b-5fddb77161d0.png)